.travis.yml

```diff
@@ -34,6 +34,7 @@ install:
     - ${TRAVIS_BUILD_DIR}/install.sh
 
     # Install testing dependencies
+    - pip install six
```

Comment:
@moylop260 
This change was necessary for fix the errors in travis
https://travis-ci.org/Vauxoo/pylint-odoo/jobs/235222591#L428